guide/daily-spending-wallet/backup-and-recovery/landing-page.md

```diff
@@ -50,7 +50,7 @@ Two main types of information need to be backed up: the user's wallet recovery d
    alt-text = "A diagram of data required to back up a bitcoin wallet"
    width = 1200
    height = 680
-   layout = "full-width"
+  
```

Comment:
Since the layout attribute has been removed, could you remove this line entirely?

Reply:
Thank you your help @sbddesign. Do you know how I can link the image in the .md file on my end after exporting from figma?

Reply:
I think the best way to handle it is to simply save over the old image file with the same filename.

Reply:
You will need to remove the entry from the `compress_image_cache-yml` so it compresses the new image if you do that though

Reply:
Thank you